cmd/eksctl-anywhere/cmd/upgrademanagementcomponents.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/cmd/eksctl-anywhere/cmd/aflag"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/types"
+	"github.com/aws/eks-anywhere/pkg/workflows/management"
+	"github.com/spf13/cobra"
+)
+
+type upgradeManagementComponentsOptions struct {
+	clusterOptions
+}
+
+var umco = &upgradeManagementComponentsOptions{}
+
+func init() {
+	flagSet := upgradeManagementComponentsCmd.Flags()
+	aflag.String(aflag.ClusterConfig, &umco.fileName, flagSet)
+	aflag.String(aflag.BundleOverride, &umco.bundlesOverride, flagSet)
+}
+
+var upgradeManagementComponentsCmd = &cobra.Command{
+	Use:          "management-components",
+	Short:        "Upgrade management components in a management cluster",
+	Long:         "This command is used to upgrade management components in a management cluster",
+	PreRunE:      bindFlagsToViper,
```

Comment:
nit: are we using viper?

Reply:
It's there because consistency with other commands and we need "viper.AutomaticEnv()" inside "bindFlagsToViper".